packages/conf-binutils/conf-binutils.master/files/find-binutils.ml.in

```diff
@@ -0,0 +1,228 @@
+open Printf
+open StdLabels
+
+type file = {
+  path : string;
+  dgst : string;
+}
+
+type tool = {
+  test  : string -> bool;
+  main  : file option;
+  files : file list;
+}
+
+module Digests = Set.Make(String)
+module Vars = Map.Make(String)
```

Comment:
```suggestion
module Names = Map.Make(String)
```

and substitute everything below. The key is not really a variable but a name of a tool.